### PR TITLE
Allow org admins with FORCE_EDIT_LIBRARIES to move libraries as if they own them

### PR DESCRIPTION
**Pull request by ryanpbrewster**
Don't merge until you backfill REMOVE_LIBRARY permissions

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/model/Organization.scala [hunk 1]
```diff
@@ -88,11 +88,21 @@ object Organization extends ModelWithPublicIdCompanion[Organization] {
...
+        ADD_LIBRARIES,
+        REMOVE_LIBRARIES
+      ),
       Some(OrganizationRole.MEMBER) -> Set(
-        OrganizationPermission.VIEW_ORGANIZATION,
-        OrganizationPermission.ADD_LIBRARIES
+        VIEW_ORGANIZATION,
+        ADD_LIBRARIES,
+        REMOVE_LIBRARIES,
+        INVITE_MEMBERS
```
Comment (ryanpbrewster) [inline kifi-backend/modules/common/app/com/keepit/model/Organization.scala]: Do people have opinions about this?

**Comment by ryanpbrewster** [comment]
@stkem @leogrim since I've already bugged y'all about permissions you should know about this.

File: kifi-backend/modules/common/app/com/keepit/model/Organization.scala (hunk 1 above)
Comment (stkem) [inline kifi-backend/modules/common/app/com/keepit/model/Organization.scala]: I somewhat prefer the former style, but don't feel super strongly.

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala [hunk 2]
```diff
@@ -494,23 +486,25 @@ class LibraryCommanderImpl @Inject() (
...
       val from: LibrarySpace = library.space
-      val userOwnsLibrary = library.ownerId == userId
       val canMoveFromSpace = from match {
         case OrganizationSpace(fromOrg) =>
-          library.ownerId == userId
-        // TODO(ryan): when the frontend has UI for this, add it in
-        // organizationMembershipCommander.getPermissions(fromOrg, Some(userId)).contains(OrganizationPermission.REMOVE_LIBRARIES)
-        case UserSpace(fromUser) => fromUser == userId // Can move libraries from Personal space to Organization Space.
+          val fromPermissions = orgMembershipCommander.getPermissionsHelper(fromOrg, Some(userId))
+          fromPermissions.contains(FORCE_EDIT_LIBRARIES) || (userId == library.ownerId && fromPermissions.contains(REMOVE_LIBRARIES))
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala:497]: @stkem here

Comment (stkem) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala:497]: Thanks!

**Comment by stkem** [comment]
:+1:

**Pull request merged**